grayscale.py

```diff
@@ -208,21 +225,16 @@ def __init__(self):
         self.fill(Grayscale.BLACK)
         self._copy_buffers()
         self.init_display()
+        gc.collect()
```

Comment:
Is there a particular reason for this `gc.collect`? So we have a better chance at allocating memory for the thread initialization?

Reply:
I think this is just a holdover from when the GPU thread was doing `gc.collect()` on each iteration. I'll remove it.